src/main/java/loginlogout/LoginServlet.java

```diff
@@ -0,0 +1,41 @@
+package loginlogout;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+
+import javax.servlet.ServletException;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.*;
+
+/**
+ * Allows Customer enter to system and see information.
+ * Now to enter you can input any name and password 123.
+ */
+@WebServlet("/login")
+public class LoginServlet extends HttpServlet {
+
+    @Override
+    protected void doGet(HttpServletRequest req, HttpServletResponse resp) throws ServletException, IOException {
+        req.getRequestDispatcher("login.jsp").forward(req, resp);
+    }
+
+    @Override
+    protected void doPost(HttpServletRequest request, HttpServletResponse response) throws ServletException, IOException {
+        response.setContentType("text/html");
+        PrintWriter out = response.getWriter();
```

Comment:
is it possible to set response properties closer to end of the method?

Reply:
What do you mean? 0_o